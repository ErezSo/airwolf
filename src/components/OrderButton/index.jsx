import React from "react";

const styles = {
  buttonBox: {
    position: "absolute",
    width: "100%",
    height: "100%",
    zIndex: "10",
    display: "flex",
    justifyContent: "center",
    alignItems: "center",
    cursor: "pointer"
  },
  orderButton: {
    width: 139,
    height: 33,
    border: "2px solid black",
    fontSize: 13,
    display: "flex",
    justifyContent: "center",
    alignItems: "center",
    fontWeight: 600
  }
};

function OrderButton({ containerStyle, reserveItem }) {
  const { buttonBox, orderButton } = styles;
  return (
    <div style={containerStyle}>
      <div style={buttonBox} onClick={reserveItem}>
        <div style={orderButton}>Add to Cart</div>
      </div>
    </div>
  );
}

export default OrderButton;
