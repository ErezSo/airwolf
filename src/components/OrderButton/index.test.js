import React from "react";
import { shallow } from "enzyme";
import OrderButton from "./";

describe("OrderButton", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<OrderButton />);
  });

  it("renders", () => {
    expect(wrapper).toHaveLength(1);
  });

  it("renders correctly", () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
