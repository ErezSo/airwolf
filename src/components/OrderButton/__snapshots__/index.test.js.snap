// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderButton renders correctly 1`] = `
"<div style={[undefined]}>
  <div style={{...}} onClick={[undefined]}>
    <div style={{...}}>
      Add to Cart
    </div>
  </div>
</div>"
`;
